### PR TITLE
Background color fix for replies. (Normally) fixes #77, fixes #81 and…

### LINKED ISSUE #77: Whatsapp background has black bars on my messages
Hey, I got this issue, that my messages got black bars on the background.
It looks like this: 
image

Now I don't know if there is something wrong with my whatsapp or not, but any advice would help.

Thanks for helping!

### LINKED ISSUE #80: Theme resets on every restart - MacOS Catalina Issue
I install the theme. Restart my Mac, and the theme settings do not persist after the reboot of the machine. If I try to re-install the theme after it resets, then it messes the theme up(Grey patches around the message blocks) with inconsistent styling.

### LINKED ISSUE #81: Dark lines in the Chat
image

i tested every theme and the line is still there

### PR DESCRIPTION
**Pull request by haveneersrobin**
Background color fix for replies. (Normally) fixes #77, fixes #81 and fixes #80

### CODE REVIEW

**Review by insign — APPROVED** [review]
Worked great for me and the telegram theme, osx

**Comment by lukassr** [comment]
It works perfectly. Please merge.

**Comment by joaoavilars** [comment]
Perfect!

**Comment by MiLeX84** [comment]
Background is fixed but new issue due to this change with chat bubbles. 
The chat bubbles are now in original color theme with bright text on the bright green and grey original bubble colors.
Only the bubbles are in original color, the text is to bright, its a very light grey, unpleasant to read.
The bubble option menu is black when hovering over it with the mouse, the menu itself is fine. 

WhatApp: 0.3.9309
Windows10: 1909 - 18363.535

WhatsApp
2020-01-04 14_35_13-WhatsApp
WADark-Windows_styles_win32_dark css - Sublime Text (UN

File: styles/win32/dark.css [hunk 1]
```diff
@@ -602,17 +602,12 @@ html[dir=ltr] ._3kx-g:hover:not(._3E86i) {
...
-html[dir] .message-in .Nxkz1, .message-out, .message {
-    background-color: var(--darker) !important; }
-html[dir] .message-out .Nxkz1 {
-    background-color: var(--msgout) !important; }
-/* Deleted messages */
-._3hePT, ._2bkkE {
-    color: var(--light) !important;
-    opacity: 0.70;
+/* Message bubbles. */ 
+html[dir] .message-in ._2Wx_5 {
```
Comment (alistair-low) [inline styles/win32/dark.css:606]: This is the wrong class name on Windows, it should be the original .Nxkz1.

File: styles/win32/dark.css [hunk 2]
```diff
@@ -602,17 +602,12 @@ html[dir=ltr] ._3kx-g:hover:not(._3E86i) {
...
+    background-color: var(--darker) !important;
+} 
+html[dir] .message-out ._2Wx_5 {
```
Comment (alistair-low) [inline styles/win32/dark.css:609]: As on line 606, the class name should be .Nxkz1.

**Comment by alistair-low** [comment]
@MiLeX84 this is the correct code on Windows:
```
/* Message bubbles. */
html[dir] .message-in .Nxkz1 {
    background-color: var(--darker) !important; }
html[dir] .message-out .Nxkz1 {
    background-color: var(--msgout) !important; }
}
```

**Comment by haveneersrobin** [comment]
I was trying to test this out once more and include the right code for Windows that @alistair-low suggested, but running the WADark script corrupts my WhatsApp application. It starts with the message 'Couldn't Update Automatically' and updating makes no difference. Whenever I run the WADark script, the application becomes unusable. I will investigate but maybe someone else has an idea.

image

**Comment by m4heshd** [comment]
Hi @haveneersrobin,
This PR is great but it was already fixed in the original local repo. All of you just have to wait patiently for the next release because everything is already fixed. I'm just running the final tests and finishing up documentation. Meanwhile I highly recommend you all not to use 3rd party style sheets or scripts because you might end up getting banned. Way i do this, it's really really careful not to interrupt the app functionality.

Even though this PR won't get merged, always feel free to join the development and contribute. Also make sure to join the telegram chat and introduce yourself. I'm creating a team of maintainers. Thank you for your contribution.

**Comment by haveneersrobin** [comment]
> Hi @haveneersrobin,
> This PR is great but it was already fixed in the original local repo. All of you just have to wait patiently for the next release because everything is already fixed. I'm just running the final tests and finishing up documentation. Meanwhile I highly recommend you all not to use 3rd party style sheets or scripts because you might end up getting banned. Way i do this, it's really really careful not to interrupt the app functionality.
> 
> Even though this PR won't get merged, always feel free to join the development and contribute. Also make sure to join the telegram chat and introduce yourself. I'm creating a team of maintainers. Thank you for your contribution.

Great to hear @m4heshd !

**Pull request closed without merging**